examples/reactjs/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
In Mardown, I think you don't need to escape the newline.

Reply:
It's the default README of create-react-app :-) Will remove this.